docs/algorithms/kem/saber.yml

```diff
@@ -44,6 +44,18 @@ parameter-sets:
     no-secret-dependent-branching-checked-by-valgrind: true
     large-stack-usage: false
     upstream: primary-upstream
+  - upstream-id: aarch64
+    supported-platforms:
+    - architecture: ARM64_V8
+      operating_systems:
+      - Linux
+      - Darwin
+    common-crypto:
+    - SHA3: liboqs
+    no-secret-dependent-branching-claimed: false
+    no-secret-dependent-branching-checked-by-valgrind: true
```

Comment:
How can it be that there's it is not claiming no-secret-dependent-branching but we're checking it anyway?

Reply:
Oops; indeed: We don't have tests: #1119 . This must be corrected. Reverted my "LGTM" above :)

Reply:
I've updated the aarch64 no-secret-dependent-branching-checked-by-valgrind to false, however I noticed it on the avx2 implementations as well. Is that intentional?

Reply:
> Is that intentional?

I can't answer that: Just checked all "no-secret-dependent-branching-checked-by-valgrind" in main branch and they're _all_ `true`, so any change to this value must be a result of the latest run of `update_docs_from_yaml.py`... Also see https://github.com/open-quantum-safe/liboqs/issues/1114

Reply:
We do have a [CI job](https://github.com/open-quantum-safe/liboqs/blob/f0678827043112e4c2f3854f40a1258180a0665b/.circleci/config.yml#L367) that runs the constant-time analysis on Haswell which includes AVX2.